be/src/exec/pipeline/pipeline_driver_dispatcher.cpp

```diff
@@ -8,11 +8,16 @@
 
 namespace starrocks::pipeline {
 
-GlobalDriverDispatcher::GlobalDriverDispatcher(std::unique_ptr<ThreadPool> thread_pool)
-        : _driver_queue(std::make_unique<DriverQueueWithWorkGroup>()),
-          _thread_pool(std::move(thread_pool)),
+GlobalDriverDispatcher::GlobalDriverDispatcher(std::unique_ptr<ThreadPool> thread_pool, bool is_low_latency)
+        : _thread_pool(std::move(thread_pool)),
           _blocked_driver_poller(new PipelineDriverPoller(_driver_queue.get())),
-          _exec_state_reporter(new ExecStateReporter()) {}
+          _exec_state_reporter(new ExecStateReporter()) {
+    if (is_low_latency) {
+        _driver_queue = std::make_unique<QuerySharedDriverQueue>();
```

Comment:
`QuerySharedDriverQueue` in the workgroup_main branch is different from that in the main branch.
Maybe we should rename `QuerySharedDriverQueue` in the workgroup_main branch to something like `DriverQueueWithoutLock` and copy ``QuerySharedDriverQueue` in the main branch to workgroup_main.